assets/css/legend.scss

```diff
@@ -0,0 +1,118 @@
+@media screen and (max-width: 450px) {
```

Comment:
you can define all media query rules inside the default ones : http://thesassway.com/intermediate/responsive-web-design-in-sass-using-media-queries-in-sass-32